applicationinsights/applicationinsights-js.d.ts

```diff
@@ -0,0 +1,488 @@
+// Type definitions for ApplicationInsights-JS v0.22.14
+// Project: https://github.com/Microsoft/ApplicationInsights-JS
```

Comment:
ah, okay. please use `applicationinsights-js/applicationinsights-js.d.ts` instead of `applicationinsights/applicationinsights-js.d.ts`